runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -87,6 +87,8 @@
 	BOSS_ZALCANO("Zalcano", DiscordAreaType.BOSSES, 13250),
 	BOSS_ZULRAH("Zulrah", DiscordAreaType.BOSSES, 9007),
 	BOSS_NIGHTMARE("Nightmare of Ashihama", DiscordAreaType.BOSSES, 15515),
+	// KBD shares its region with NMZ and should be manually checked for and assigned by the plugin
+	BOSS_KING_BLACK_DRAGON("King black dragon", DiscordAreaType.BOSSES),
```

Comment:
Given that we don't show players as being in the wilderness and players necessarily must travel through the wilderness to get to KBD, I think we shouldn't show this at all.